src/Docker/Runner/StateFile.php

```diff
@@ -136,7 +142,28 @@ public function persistState(
                     ],
                 ],
             ];
-            $this->logger->notice('Storing state: ' . json_encode($state));
+            $this->logger->notice('Storing state: ' . json_encode($jobState));
```

Comment:
Tady jsme si rikal, ze by bylo dobry porad logovat jen tu cast state, ktera je relevantni k jobu, ne cely.

Reply:
já bych ho možná nelogoval vůbec, od toho se zapisuje do konfigurace...

Reply:
taky me to napadlo, ale nechtelo se mi do toho zbytecne stourat :slightly_smiling_face: